src/gg/codie/mineonline/plugin/bukkit/MineOnlineConfig.java

```diff
@@ -0,0 +1,87 @@
+package gg.codie.mineonline.plugin.bukkit;
+
+import org.bukkit.util.config.Configuration;
+
+import java.io.File;
+
+public class MineOnlineConfig extends Configuration {
+
+
+    public MineOnlineConfig(File file) {
+        super(file);
+        this.reload();
+    }
+
+    private void write() {
+        //Main
+        generateConfigOption("config-version", 1);
+        //Setting
+        generateConfigOption("public", true);
```

Comment:
Please keep max-players, whitelist and online-mode in the server.properties file.